src/evict/evict_page.c

```diff
@@ -646,41 +646,39 @@ __evict_review(
 
 	if (closing)
 		LF_SET(WT_REC_VISIBILITY_ERR);
-	else if (!WT_PAGE_IS_INTERNAL(page) &&
-	    !F_ISSET(S2BT(session), WT_BTREE_LOOKASIDE)) {
-		if (F_ISSET(conn, WT_CONN_IN_MEMORY))
-			LF_SET(WT_REC_IN_MEMORY |
-			    WT_REC_SCRUB | WT_REC_UPDATE_RESTORE);
-		else if (WT_SESSION_BTREE_SYNC(session))
-			LF_SET(WT_REC_LOOKASIDE);
-		else if (!WT_IS_METADATA(session->dhandle)) {
```

Comment:
The core change is moving this test down to line 673.  The rest is reformatting.